receiver/oracledbreceiver/metadata.yaml

```diff
@@ -13,6 +13,54 @@ resource_attributes:
     description: The name of the instance that data is coming from.
     enabled: true
     type: string
+  oracledb.username:
```

Comment:
as the metadata file only support metrics now, i personally think add those attribute would be confusing to enduser.

Reply:
so firstly i think there is no logic in the code on how you are controlling if the user would get those attributes.
secondly, if we have those in the metadata file, people would expect to see those attributes in the exported metrics as well, but eventually they will not see it. they will be confused. 
thirdly, i am not sure if the user need to control what attributes to be exposed.